{"uid":"a86ef87d29e437f7","name":"test_failed","fullName":"test.allure_status_test#test_failed","historyId":"4da3226ec9761b158ba5f826fc8f8289","time":{"start":1674029138497,"stop":1674029138497,"duration":0},"status":"failed","statusMessage":"AssertionError: Some fail reason\nassert False","statusTrace":"def test_failed():\n>       assert False, \"Some fail reason\"\nE       AssertionError: Some fail reason\nE       assert False\n\ntest/allure_status_test.py:9: AssertionError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"afterStages":[],"labels":[{"name":"parentSuite","value":"test"},{"name":"suite","value":"allure_status_test"},{"name":"host","value":"fv-az461-792"},{"name":"thread","value":"2264-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"test.allure_status_test"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"a86ef87d29e437f7.json","parameterValues":[]}